### ISSUE TITLE
Usage example does not appear to be working.

### ISSUE THREAD

**Issue by mmwtsn** [yatish27/linkedin-scraper#31]
When walking through the usage instructions contained within the README, nearly all of the suggested methods were returning `nil` or an empty array.

I was having issues scraping LinkedIn using Nokogiri and Ruby's OpenURI module before trying out this gem. Perhaps LinkedIn is doing something to interfere with scraping attempts?

Tested using the following:

```
linkedin-scraper (0.1.2)
ruby 2.1.1p76 (2014-02-24 revision 45161) [x86_64-darwin12.0]
```

**Comment by jtoy** [comment]
same issue,does this work?

**Comment by yatish27** [comment]
Fixed 0.1.5
